docs/dev/table/connect.md

```diff
@@ -1321,12 +1322,12 @@ The CSV format can be used as follows:
 .withFormat(
   new Csv()
 
-    // required: define the schema either by using type information
-    .schema(Type.ROW(...))
-
-    // or use the table's schema
+    // optional: use the table's schema as format schema, this is enabled by default
     .deriveSchema()
```

Comment:
Maybe we can remove it from doc now?
Including `.deriveSchema()` and `.schema(Type.ROW(...))`?

Reply:
I agree to remove deriveSchema, but should still keep `.schema(Tyeps.ROW(...))`, because it is not dropped as we discussed.